bubbles/commands/isup.py

```diff
@@ -5,24 +5,20 @@
 from bubbles.service_utils import SERVICES, get_service_name
 
 
-def isup(payload: Payload):
-    """!isup [service_name] - is the service currently running?"""
+def isup(payload: Payload) -> None:
+    """!isup [service_name] - is the service currently running?."""
```

Comment:
lol wat

Reply:
lol, this auto fix is definitely super reliable.
I'll fix it tomorrow